system-parachains/bridge-hubs/bridge-hub-kusama/src/bridge_to_polkadot_config.rs

```diff
@@ -231,24 +242,31 @@ type FromPolkadotMessageBlobDispatcher = BridgeBlobDispatcher<
 >;
 
 /// Export XCM messages to be relayed to the other side
-pub type ToBridgeHubPolkadotHaulBlobExporter = HaulBlobExporter<
-	XcmBlobHaulerAdapter<ToBridgeHubPolkadotXcmBlobHauler>,
-	PolkadotGlobalConsensusNetwork,
-	(),
->;
+pub type ToBridgeHubPolkadotHaulBlobExporter = XcmOverBridgeHubPolkadot;
 pub struct ToBridgeHubPolkadotXcmBlobHauler;
 impl XcmBlobHauler for ToBridgeHubPolkadotXcmBlobHauler {
 	type Runtime = Runtime;
 	type MessagesInstance = WithBridgeHubPolkadotMessagesInstance;
-	type SenderAndLane = FromAssetHubKusamaToAssetHubPolkadotRoute;
 
 	type ToSourceChainSender = XcmRouter;
 	type CongestedMessage = bp_asset_hub_kusama::CongestedMessage;
 	type UncongestedMessage = bp_asset_hub_kusama::UncongestedMessage;
 }
 
+/// Add support for the export and dispatch of XCM programs.
+pub type XcmOverBridgeHubPolkadotInstance = pallet_xcm_bridge_hub::Instance1;
+impl pallet_xcm_bridge_hub::Config<XcmOverBridgeHubPolkadotInstance> for Runtime {
+	type UniversalLocation = UniversalLocation;
+	type BridgedNetworkId = PolkadotGlobalConsensusNetwork;
+	type BridgeMessagesPalletInstance = WithBridgeHubPolkadotMessagesInstance;
+	type MessageExportPrice = ();
```

Comment:
Dont know what this means, but why is the price zero?

Reply:
`MessageExportPrice` is just propagated to the [`xcm_builder::HaulBlobExporter`](https://github.com/paritytech/polkadot-sdk/blob/release-polkadot-v1.5.0/polkadot/xcm/xcm-builder/src/universal_exports.rs#L425-L429) component [here](https://github.com/paritytech/polkadot-sdk/blob/release-polkadot-v1.5.0/bridges/modules/xcm-bridge-hub/src/exporter.rs#L34-L37), and we don't need extra additional price here, because actual weights/price is covered by XCM `ExportMessage` instruction and  benchmark

Reply:
Let's add a code comment explaining this

Reply:
added here: https://github.com/polkadot-fellows/runtimes/pull/137/commits/f82cc0536f5b4c272cea185f3e31d2ca9d9537df